system/file_windows.go

```diff
@@ -0,0 +1,31 @@
+package system
+
+import (
```

Comment:
Does this need a //build +windows ? Am unfamiliar with conditional compilation.

Reply:
Nope, the `_windows` suffix implies it's only for windows. If this did not work, travis would fail.